src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_validators.py

```diff
@@ -219,7 +220,8 @@ def test_validate_with_source_uri_blob(self):
         # source given in form of uri, source_container parsed from uri, source and dest account are the same
         ns = self._create_namespace(source_uri='https://storage_name.blob.core.windows.net/container2',
                                     destination_container='container1')
-        get_source_file_or_blob_service_client(MockCmd(self.cli), ns)
+        with mock.patch('azure.cli.command_modules.storage._validators._query_account_key', return_value="fake_key"):
```

Comment:
It would be nice to add some test coverage for how the validators handle using a source_uri with a source_sas, if possible.